### PR TITLE
Update notebooks to use grpc4bmi==0.2.12 api

### PR DESCRIPTION
**Pull request by sverhoeven**
See https://github.com/eWaterCycle/grpc4bmi/pull/94

### CODE REVIEW

File: walrus/walrus-bmi-psfetch.ipynb [hunk 1]
```diff
@@ -57,17 +62,17 @@
...
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 21,
    "metadata": {},
    "outputs": [],
    "source": [
-    "model = BmiClientDocker(image=models['WALRUS']['docker'], image_port=55555, \n",
-    "                        input_dir=\"./input\")"
+    "model = BmiClientDocker(image='ewatercycle/walrus-grpc4bmi:v0.2.0', image_port=55555, \n",
+    "                        input_dirs=(\"./input\",))"
```
Comment (SarahAlidoost) [inline walrus/walrus-bmi-psfetch.ipynb:70]: Please add `work_dir` to BmiClientDocker().

**Review by SarahAlidoost — APPROVED** [review]
@sverhoeven looks good, just a minor fix for walrus notebook is needed. I left a comment.

File: walrus/walrus-bmi-psfetch.ipynb (hunk 1 above)
Comment (sverhoeven) [reply walrus/walrus-bmi-psfetch.ipynb]: done

**Comment by sverhoeven** [comment]
Thanks for reviewing, great that you noticed the missing work_dir.

**Pull request merged**